plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -336,28 +336,29 @@ public void testDropNonEmptySchemaWithTable()
     @Override
     public void testRenameColumn()
     {
-        // Override because the connector doesn't support renaming columns with 'none' column mapping
-        // There are some tests in in io.trino.tests.product.deltalake.TestDeltaLakeColumnMappingMode
-        assertThatThrownBy(super::testRenameColumn)
-                .hasMessageContaining("Cannot rename column with the column mapping: NONE");
+        // Override because the default column mapping mode 'none' doesn't support renaming columns
+        super.testRenameColumn("WITH (column_mapping_mode = 'id')");
+        super.testRenameColumn("WITH (column_mapping_mode = 'name')");
```

Comment:
I preferred the previous version: BCT covering the default setup and separate test methods covering the non-default configuration